### PR TITLE
SASS-702 - Added controller that returns a page to view all employmen…

### PR DESCRIPTION
**Pull request by Dula-J**
…t expenses

### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME] https://jira.tools.tax.service.gov.uk/browse/SASS-702

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [x]  Have you checked the PR Builder passes?
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

**Review by BenCosfordHMRC — APPROVED** [review]

File: it/controllers/CheckEmploymentExpensesControllerISpec.scala [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+import helpers.{PlaySessionCookieBaker, ViewTestHelper}
+import models.{ExpensesType, GetEmploymentExpensesModel}
+import org.jsoup.Jsoup
+import play.api.http.HeaderNames
+import play.api.libs.ws.{WSClient, WSResponse}
+import utils.IntegrationTest
+import play.api.http.Status._
+import play.api.libs.json.Json
+
+class CheckEmploymentExpensesControllerISpec extends IntegrationTest with ViewTestHelper {
```
Comment (BenCosfordHMRC) [inline it/controllers/CheckEmploymentExpensesControllerISpec.scala:29]: Missing controller unit tests

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

File: app/controllers/CheckEmploymentExpensesController.scala [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import models.GetEmploymentExpensesModel
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import utils.SessionHelper
+import views.html.expenses.CheckEmploymentExpensesView
+
+import javax.inject.Inject
+
+class CheckEmploymentExpensesController @Inject()(authorisedAction: AuthorisedAction,
```
Comment (BenCosfordHMRC) [inline app/controllers/CheckEmploymentExpensesController.scala:31]: can move this controller into employment folder in controllers package

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

File: app/controllers/CheckEmploymentExpensesController.scala (hunk 2 above)
Comment (Dula-J) [reply app/controllers/CheckEmploymentExpensesController.scala]: Done

File: it/controllers/CheckEmploymentExpensesControllerISpec.scala (hunk 1 above)
Comment (Dula-J) [reply it/controllers/CheckEmploymentExpensesControllerISpec.scala:29]: Added

**Review by BenCosfordHMRC — APPROVED** [review]

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**